presto-spi/src/main/java/com/facebook/presto/spi/statistics/TableStatistics.java

```diff
@@ -111,6 +121,7 @@ public static final class Builder
         private Estimate rowCount = Estimate.unknown();
         private Estimate totalSize = Estimate.unknown();
         private Map<ColumnHandle, ColumnStatistics> columnStatisticsMap = new LinkedHashMap<>();
+        private ConfidenceLevel confidenceLevel = HIGH;
```

Comment:
Why is the default high? Should we keep this to unknown or something?

Reply:
@jaystarshot Previous code had it default as true, here we are keeping it at high which is lower than fact. In cases where there is filtering it is set to low. What do you think?